### PR TITLE
Replace SendHandshake with MessageSender.SendToSpecific.

### PR DESCRIPTION
**Pull request by yoshivb**
This should resolve issue #8.

### CODE REVIEW

File: WebSocketTest/ConnectionHandlers/ClientConnection.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public void Accept(Client clientData)
 			{
 				try
 				{
-					SendHandshake(clientData, Handshake.GenerateHandshake(data));
+                    MessageSender.SendToSpecific(data, clientData);
```
Comment (Deefoozy) [inline WebSocketTest/ConnectionHandlers/ClientConnection.cs]: This replaces the generated handshake with the received request, HandShake.GenerateHandshake is still required (but has to be adapted to return the string it encodes by removing the utf8encoding at the end) 
```suggestion
					MessageSender.SendToSpecific(Handshake.GenerateHandshake(data), clientData);
```

**Review by Deefoozy — CHANGES_REQUESTED** [review]

File: WebSocketTest/ConnectionHandlers/ClientConnection.cs (hunk 1 above)
Comment (yoshivb) [reply WebSocketTest/ConnectionHandlers/ClientConnection.cs]: Ah yes, my bad. Working on it 👍

Comment (yoshivb) [reply WebSocketTest/ConnectionHandlers/ClientConnection.cs]: Just committed changes that should fix this.

Comment (Deefoozy) [reply WebSocketTest/ConnectionHandlers/ClientConnection.cs]: I'll review it in a moment, VS updates are taking longer than expected...

**Review by Deefoozy — APPROVED** [review]

**Comment by Deefoozy** [comment]
Thanks alot for the contribution! Have a nice day!

**Pull request merged**